cpp/tests/fixed_point/fixed_point_tests.cu

```diff
@@ -27,6 +29,11 @@
 #include <type_traits>
 #include <vector>
 
+#include <thrust/transform.h>
+#include "cudf/null_mask.hpp"
+#include "cudf/types.hpp"
+#include "cudf/utilities/error.hpp"
```

Comment:
Move these cudf includes to the _cudf_ section above.

Reply:
Done.

Reply:
And fix quotes --> angle brackets.